R/after-wrappers.R

```diff
@@ -247,7 +247,7 @@ pl$show_all_public_methods = function(class_names = NULL) {
 #' @return an element from the public namespace `pl` polars. Likely a function or an RPolarsDataType
 #' @export
 #' @noRd
-#' @keywords internal
+#' @noRd
```

Comment:
`@noRd` already exists in the line above.

Reply:
It's annoying to go through all files to check this and it doesn't warn or error so let's just remove the duplicate when we come across one